tests/test_frame.py

```diff
@@ -1174,10 +1229,13 @@ def test_types_groupby() -> None:
             upper="2.2.99",
         ):
             if PD_LTE_22:
-                df7: pd.DataFrame = df.groupby(by="col1").apply(sum)
-    df8: pd.DataFrame = df.groupby("col1").transform("sum")
+                check(
+                    assert_type(df.groupby(by="col1").apply(sum), pd.DataFrame),
+                    pd.DataFrame,
+                )
+    check(assert_type(df.groupby("col1").transform("sum"), pd.DataFrame), pd.DataFrame)
     s1: pd.Series = df.set_index("col1")["col2"]
```

Comment:
why not convert this line?

Reply:
That is something I missed, I was going by "every time we define a variable that we don't reuse", in that case `s1` was reused below.